README.md

```diff
@@ -197,7 +197,10 @@ helmfiles:
   # Override state values
   values:
   # Values files merged into the nested state's values
-  - additiona.values.yaml
+  - additional.values.yaml
+  # One important aspect of using values here is that they first need to be defined in the releases section
+  # of the origin helmfile, so in this example key1 needs to be in the releases section of the chart you
```

Comment:
This overrides state values rather than release values so the sentence should be like "key1 needs to be in the `values` or `environments.NAME.values` of `path/to/subhelmfile.yaml`"

Reply:
@mumoshu made the change, let me know if it makes sense

Reply:
Thanks! One more comment and this seems like good to go